### DISCUSSION TITLE
关于蓝盾流水线中部分超链接不能通过点击打开的问题

### DISCUSSION THREAD

**Discussion by 29797471** [Tencent/bk-ci#6124] [General]
> 目前有遇到蓝盾流水线中,一些自定义的超链接(地址:file://tencent.com/Tfs/XXX/XXX/xx.txt )不能通过点击打开,右键->在新窗口打开链接后访问的地址是about:blank#blocked,但右键->复制链接,再在新网页粘贴这个链接地址后可以正常打开.个人猜测是流水线直接屏蔽了所有tfs访问,建议如果用户有正常访问这个流水线的权限,那么应该可以通过网页触发访问这个链接,而这个链接能否正常打开,交由tfs文件系统来判定.

**Comment by iccengan** [comment]
哪个地方展示的超链接？插件日志里面吗？

**Reply by 29797471** [reply]
是的
